akka-cluster-sharding/src/main/scala/akka/cluster/sharding/internal/RememberEntitiesStarter.scala

```diff
@@ -30,7 +33,8 @@ private[akka] object RememberEntityStarter {
       settings: ClusterShardingSettings) =
     Props(new RememberEntityStarter(region, shard, shardId, ids, settings))
 
-  private case object Tick extends NoSerializationVerificationNeeded
+  private final case class StartBatch(batchSize: Int) extends NoSerializationVerificationNeeded
```

Comment:
if the batch size is always the same can this be a case object

Reply:
It's from user config